<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>project_window_4</class>
 <widget class="QDialog" name="project_window_4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>message</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(171, 171, 171);
background-color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>doing my tasks</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(171, 171, 171);
background-color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>doing my tasks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>overall status</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(171, 171, 171);
background-color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>overall status</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>all members</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(200, 171, 100);
background-color: rgb(50, 50, 50);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
